lookout/style/format/rules.py

```diff
@@ -134,13 +140,17 @@ def predict(self, X: numpy.ndarray, vnodes_y: Sequence[VirtualNode],
             postprocess = import_module(
                 "lookout.style.format.langs.%s.postprocessor" % feature_extractor.language) \
                 .postprocess
+            filter_uast_breaking_preds = import_module(
+                "lookout.style.format.postprocess").filter_uast_breaking_preds
         except ImportError:
             return y_pred, winners
-        postprocessed_y, postprocessed_winners = postprocess(X, y_pred, vnodes_y, vnodes, winners,
-                                                             self, feature_extractor)
+        postprocessed_y_pred, postprocessed_winners = postprocess(X, y_pred, vnodes_y, vnodes,
```

Comment:
the same suggestion - named arguments to function when number of arguments is quite big - it could easier our pain in future